corehq/messaging/smsbackends/http/models.py

```diff
@@ -73,25 +73,39 @@ def send(self, msg, *args, **kwargs):
         verify_sms_url(config.url, msg, backend=self)
 
         url_params = urlencode(params)
+        ssl_context = self.create_ssl_context()
         try:
-            unverified = ssl._create_unverified_context()
             if config.method == "GET":
                 urlopen(
                     "%s?%s" % (config.url, url_params),
-                    context=unverified,
+                    context=ssl_context,
                     timeout=settings.SMS_GATEWAY_TIMEOUT,
                 ).read()
             else:
                 urlopen(
                     config.url,
                     url_params,
-                    context=unverified,
+                    context=ssl_context,
                     timeout=settings.SMS_GATEWAY_TIMEOUT,
                 ).read()
         except Exception as e:
             msg = "Error sending message from backend: '{}'\n\n{}".format(self.pk, str(e))
             six.reraise(BackendProcessingException, BackendProcessingException(msg), sys.exc_info()[2])
 
+    def create_ssl_context(self):
+        # Protocol and ciphers are explicitly set for backwards compatibility reasons with older versions
+        # of OpenSSL. Clients still using older SSL versions will break without these changes.
+        # See: https://github.com/curl/curl/issues/10002#issuecomment-1331413797
+        context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_1)
```

Comment:
Offlined with Matt on some more detailed spec notes, but we definitely need to make it the case that the default connection used won't be an explicit downgrade to a deprecated protocol. I'm comfortable with partners having the option to allow a TLS1.1 endpoint, but it shouldn't downgrade the connection for other users.